### DISCUSSION TITLE
Beta styles

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4136] [Feature Suggestions]
I saw in the valve videos portal 2 old beta style (like pneumatic diversity vent trailer)
and I think this style would be cool

**Comment by FrozenDragon0** [comment]
I've suggested this style years ago, we aren't adding any beta content, you are welcome to make a UCP with the style however.

**Reply by SimpleFoxOfficial** [reply]
what is UCP

**Reply by slambo311** [reply]
UCPs are basically just custom content.
It stands for User Generated Packages.
